package.json

```diff
@@ -26,7 +26,8 @@
     "semver": "5.1.0",
     "underscore": "1.8.3",
     "xml2js": "0.1.13",
-    "@braintree/wrap-promise": "1.1.1"
+    "@braintree/wrap-promise": "1.1.1",
+    "js2xmlparser": "^3.0.0"
```

Comment:
TBH, I'm wary of adding an additional dependency on this library. I understand the convenience of including this, but I'm not certain it's worth the burden of including an additional external dependency.

Also, we'll want to pin to a specific version, not a floating version.